<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CryptoHelper</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="2,0,1">
      <item>
       <layout class="QHBoxLayout" name="codeInputLayout"/>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="codeOutLayout">
        <item>
         <widget class="QTextEdit" name="codeOutTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Ciphers</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QComboBox" name="ciphiersComboBox">
             <item>
              <property name="text">
               <string>Magic square cipher</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Caesar cipher</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Caesar cipher (affine)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Caesar cipher (keyword)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Trithemius cipher</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Playfair cipher</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Vigenere cipher</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Wheatstone cipher</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="encryptPushButton">
             <property name="text">
              <string>Encrypt</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Control</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QCheckBox" name="trainingModeCheckBox">
             <property name="text">
              <string>Learning mode</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="generalMenu">
    <property name="title">
     <string>General</string>
    </property>
    <addaction name="action_saveToFile"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="settingsMenu">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="action_settings"/>
   </widget>
   <widget class="QMenu" name="aboutMenu">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="action_aboutQt"/>
    <addaction name="action_aboutProgram"/>
   </widget>
   <addaction name="generalMenu"/>
   <addaction name="settingsMenu"/>
   <addaction name="aboutMenu"/>
  </widget>
  <action name="action_saveToFile">
   <property name="text">
    <string>Save to file...</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_aboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="action_aboutProgram">
   <property name="text">
    <string>About program...</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
